### PR TITLE
Update Dumbledraw with Python3

### PR DESCRIPTION
**Pull request by harrypuuter**
This PR changes Dumbledraw to run with python3 (finally)

### CODE REVIEW

File: Dumbledraw/labels.yaml
Comment (a-monsch) [inline Dumbledraw/labels.yaml:1]: Would it be beneficial to add a variable of total yield here?

Comment (harrypuuter) [reply Dumbledraw/labels.yaml:1]: yes, this would make sense !

Comment (harrypuuter) [reply Dumbledraw/labels.yaml:1]: how do you call the variable normally?

Comment (a-monsch) [reply Dumbledraw/labels.yaml:1]: For the naming "Total yield" would be sufficient and in line with the other names I guess?

**Review by a-monsch — COMMENTED** [review]
Regarding the non-formatting changes, it seems good to me...

(ongoing total yield discussion)

File: Dumbledraw/labels.yaml
Comment (a-monsch) [reply Dumbledraw/labels.yaml:1]: Thanks for adding it

**Review by a-monsch — APPROVED** [review]

**Pull request merged**